packages/core/test/Cardano/types/Address.test.ts

```diff
@@ -36,4 +36,10 @@ describe('Cardano/types/Address', () => {
   it('Address() accepts a valid testnet single address', () => {
     expect(() => Cardano.Address('addr_test1vrdkagyspkmt96k6z87rnt9dzzy8mlcex7awjymm8wx434q837u24')).not.toThrow();
   });
+
+  it('Address() throws error when stake address', () => {
```

Comment:
Also add tests for:
- Daedalus Byron address `DdzFFzCqrht4PWfBGtmrQz4x1GkZHYLVGbK7aaBkjWxujxzz3L5GxCgPiTsks5RjUr3yX9KvwKjNJBt7ZzPCmS3fUQrGeRvo9Y1YBQKQ`
- Yoroi Byron address `Ae2tdPwUPEZKkyZinWnudbNtHQddCyc6bCwLVoQx4GfH1NvwztGRpRkewTe`